draft-ietf-httpbis-unprompted-auth.md

```diff
@@ -109,129 +109,73 @@ This document uses the following terminology from {{Section 3 of
-to generate a nonce which can be signed using the chosen key. The keying
-material exporter uses a label that starts with the characters
-"EXPORTER-HTTP-Unprompted-Authentication-" (see {{schemes}} for the labels and
-contexts used by each scheme). The TLS keying material exporter is used to
-generate a 32-byte key which is then used as a nonce.
+This authentication scheme is only defined for uses of HTTP with TLS
+{{!TLS=RFC8446}}. This includes any use of HTTP over TLS as typically used for
+HTTP/2 {{H2}}, or HTTP/3 {{H3}} where the transport protocol uses TLS as its
+authentication and key exchange mechanism {{?QUIC-TLS=RFC9001}}.
 
 Because the TLS keying material exporter is only secure for authentication when
-it is uniquely bound to the TLS session {{!RFC7627}}, the Signature and HMAC
-authentication schemes require either one of the following properties:
+it is uniquely bound to the TLS session {{!RFC7627}}, the Signature
+authentication scheme requires either one of the following properties:
 
 * The TLS version in use is greater or equal to 1.3 {{TLS}}.
 * The TLS version in use is greater or equal to 1.2 and the Extended
 Master Secret extension {{RFC7627}} has been negotiated.
 
-Clients MUST NOT use the Signature and HMAC authentication
-schemes on connections that do not meet one of the two properties
-above. If a server receives a request that uses these authentication
-schemes on a connection that meets neither of the above properties,
-the server MUST treat the request as malformed.
+Clients MUST NOT use the Signature authentication scheme on connections that do
+not meet one of the two properties above. If a server receives a request that
+uses this authentication scheme on a connection that meets neither of the above
+properties, the server MUST treat the request as malformed.
 
-# Header Field Definition {#header-definition}
+# Computing the Authentication Proof {#compute-proof}
 
-The "Unprompted-Authentication" header field allows a user agent to authenticate
-with an origin server. The authentication is scoped to the HTTP request
-associated with this header field. The value of the Unprompted-Authentication
-header field is a credentials object, as defined in {{Section 11.4 of HTTP}}.
-Credentials contain an authentication scheme followed by optional authentication
-parameters.
+The user agent leverages a TLS keying material exporter {{!KEY-EXPORT=RFC5705}}
+with the label "EXPORTER-HTTP-Signature-Authentication" to generate a 32-byte
+symmetric key. That symmetric key is then used as nonce which can be signed
+using the client's chosen asymmetric private key. The resulting signature is
+then transmitted to the server using the Authorization field.
 
 # Authentication Parameters
 
-This specification defines the following authentication parameters, they can be
-used by the authentication schemes defined in {{schemes}}.
+This specification defines the following authentication parameters:
```

Comment:
You are using structured fields for the definitions of these, even though Authorization doesn't use SF.  That's good, but I might take this opportunity to make this point.

Reply:
I think I agree with the idea but I'm not sure I see what you mean. Do you have thoughts on how best to make said point?

Reply:
"These parameters use SF [RFC8941 (or -bis)] in their definition, though the Authorization field does not use SF."

Reply:
Done